lib/SPIRV/libSPIRV/SPIRVExtInst.h

```diff
@@ -221,7 +221,7 @@ SPIRV_DEF_NAMEMAP(OCLExtOpKind, OCLExtOpMap)
 typedef SPIRVDebug::Instruction SPIRVDebugExtOpKind;
 template <> inline void SPIRVMap<SPIRVDebugExtOpKind, std::string>::init() {
   add(SPIRVDebug::DebugInfoNone, "DebugInfoNone");
-  add(SPIRVDebug::CompilationUnit, "DebugCompileUnit");
+  add(SPIRVDebug::CompilationUnit, "DebugCompilationUnit");
```

Comment:
I found one more typo in debug info, I guess there could be some more. I want to prepare a separate patch for typos fix.
Do you mind removing this from the PR, and/or to place it in a separate PR?

Reply:
Removed